### ISSUE TITLE
Text file not being sent to my SMTP?

### ISSUE THREAD

**Issue by macgeown** [PandaPoodle/Rubber-Ducky-Keylogger-and-Email#1]
I have tested this on my Raspberry Pi Pico, and for the most part everything goes smoothly and appears as if it works, because I get the "Test" email from my "smtp.gmail.com" server, but I don't receive the Keylogger.txt file in the email, nor can I find one at "C:\temp\Keylogger.txt" at this location as stated within the scripts for the Keylogger. It also important to note that I am doing this test on a Windows 11 for VM-Workstation, so let me know if you think that could be to do with it. Thanks for your time :)

**Comment by PandaPoodle** [comment]
hmmmm that's interesting. I just tested it on my Windows 11 machine and it worked. I updated the repository so that might help it? you could copy the https://raw.githubusercontent.com/PandaPoodle/keylogger/main/Keylogg.txt?token=GHSAT0AAAAAAB23A6S5NYRYXMC35KWRD7SIY3HZUOA and paste it into a text file and edit the location where it saves the keylogger.txt file and see if that works? The place where you would change that would be at the beginning of the code and it looks like this. $path = "C:\temp\keylogger.txt" simply change the path to where it saves the keylogger.txt and hopefully that will work. If that does work you will also have the change the email script to reflect the new location of keylogger.txt. Let me know if any of that works, ill continue to work on it on my end.

**Comment by macgeown** [comment]
Found the route of the issue. The "temp" file on my Virtual Machine of Windows 11 & my physical PC is located at "C:\Users\User\AppData\Local\Temp", (User = Account Name) so it was not creating the Keylogger.txt. I have uploaded the Keylogger.PS1 to my own server and changed the location to "C:\Windows instead which is now working 100% fine. But before I did that I made a "Temp" file at the location of "C:\Temp" and it did start working then also, but for my PC's they are located where I have said, so I suspect the safer bet is where I have placed it, because everyone will have the Windows file in that location.

**Comment by PandaPoodle** [comment]
Great I'm glad its working. I updated the keylogger script to first create a folder named Windows1 at C:\Windows1, this will be the folder where the keylogger.txt will be saved. This was done to get around permissions. Certain files don't allow certain users to edit them. In my case I couldn't create the keylogger.txt file in the C:\Windows file. Thank you for your feedback, if the new version has any issues please let me know.

**Comment by macgeown** [comment]
Yeah that's way better, and probably work for everyone that way. Thanks for the help and for resolving the issues.
